### DISCUSSION TITLE
FIX: pipeline with id [xxxx.1.6.4] does not exist

### DISCUSSION THREAD

**Discussion by bdavedu** [Security-Onion-Solutions/securityonion#11138] [2.4] [answered]
Error: during the integration of sysmon logs with ElasticAgent, the error due to inexistent pipelines appears in logstash.log.

Temporary solution: clone and rename the pipelines shown in the next pictures.

![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/129211552/1a0117f3-2953-46a2-ba7d-c2111d5ce558)
![imagen](https://github.com/Security-Onion-Solutions/securityonion/assets/129211552/66470f54-206f-4f68-8dc7-0343eb8b44f4)

**Comment by dougburks** [comment, marked as answer]
@bdavedu Have you tried the hotfix we released yesterday?
https://blog.securityonion.net/2023/08/security-onion-2410-hotfix-20230821-now.html
https://docs.securityonion.net/en/2.4/release-notes.html
https://github.com/Security-Onion-Solutions/securityonion/issues/11038

**Reply by bdavedu** [reply]
I saw it but I haven't tried yet. I only saw that in hotfix the only fixed pipeline is "logs-system.syslog-1.6.4".

**Reply by dougburks** [reply]
Please try it and let us know whether or not it helps.

**Reply by bdavedu** [reply]
I have deployed the hotfix iso and everything is solved. Thank you!
